index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <div class="container">
+      <header>
+      <h1 id="title">freeCodeCamp Survey Form</h1>
+      <p id="description">Thank you for taking the time to help us improve the platform</p>
+    </header>
+<div>
+    <form id="survey-form">
+      <div>
+      <label for="name" id="name-label">Name</label>
+      <input type="text" name="name" id="name">
+      </div>
+
+<div>
+      <label for="email" id="email-label">E-Mail</label>
+      <input type="email" name="email" id="email">
+    </div>
+<div>
+      <label for="number" id="number-label">Age</label>
+      <input type="number" name="age" id="number" min="10" max="100">
+    </div>
+<div>
+      <p>Which option best describes your current role?</p>
+      <select name="role" id="dropdown">
+        <option value="Student">Student</option>
+        <option value="Full Time Job">Full Time Job</option>
+        <option value="Full Time Learner">Full Time Learner</option>
+        <option value="Prefer Not To Say">Prefer Not To Say</option>
+        <option value="Other">Other</option>
+      </select>
+    </div>
+    <div>
+      <p>Would you recommend freeCodeCamp to a friend?</p>
+      <label>
+        <input type="radio" value="definitly">Definitly
+      </label>
+      <label>
+        <input type="radio" value="maybe">Maybe
+      </label>
+      <label>
+        <input type="radio" value="notsure">Not Sure
+      </label>
+    </div>
+    <div>
+      <p>Any comments or suggestions?</p>
+      <textarea name="" id="" cols="30" rows="10" placeholder="Enter Your Comment Here..."></textarea>
+    </div>
+    <div>
+      <button type="submit" id="submit">Submit</button>
+    </div>
+    </form>
+  </div>
+    </div>
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Please reduce empty lines in your codebase to a maximum of two lines. Using at maximum of two empty lines in your codebase improves readability.